template/typescript/function/handler.ts

```diff
@@ -0,0 +1,10 @@
+import { FunctionEvent, FunctionContext, FunctionCallBack } from "./types";
```

Comment:
Do these classes get put into the function folder? ^ (they shouldn't)

We might need to expose these on `npm` and import them here instead.

Reply:
@kenfdev Do you think this would be possible?
I'm thinking that because the base template folder doesn't get copied over to the generated function folder, the types wouldn't be recognized... I haven't had time to test that theory though...
Maybe @shaunwarman or @bartsmykla would have some input as well?

Reply:
We're going to need to create a package like `@types/openfaas`. But the difference from the other templates is that one would need to `npm install` it after creating the new template. Would be better if that happens automatically.

To avoid it, we could include it in the `handler.ts` but I think that's going to be pretty ugly.

Reply:
Why don't we need an SDK for the node8-express template?

Reply:
In the `node8-express` template, the `express` part of the code is only in the template's `index.js` file. The `function/handler.js` file only has pure javascript in it. We're not using any types or classes from express.

With that template, it's up to the user to decide to also include express in their function code. Here, with the Typescript template, we're trying to use a class in both the template `index.js` _and_ the `function/handler.js` hence the necessary `npm install`. 

We could create the npm module/types and include it in the `function/package.json`, and use the new "Welcome message" feature to instruct the user that they will need to run `npm install` before using the template, but that could be missed.